crates/node/src/chain_spec.rs

```diff
@@ -58,6 +65,7 @@ pub fn development_config(enable_manual_seal: Option<bool>) -> Result<DevChainSp
         move || {
             DevGenesisExt {
                 genesis_config: testnet_genesis(
+                    madara_path_string.clone(),
```

Comment:
is there a need to clone here?

Reply:
yes because of it being inside the move it needs ownership. Alternatively I can call the to_owned

Reply:
got it, my bad